README.md

```diff
@@ -2,13 +2,16 @@
 
 This repository hosts the source code of the **http://developersdo.github.io/opensource** website.
 
+
 **Local open source communities** is an initiative to gather all possible public data about projects from the Dominican Republic in GitHub using the [GraphQL API](https://developer.github.com/v4/). Build and serve a public website showing all the local efforts that Dominicans are contributing toward open source.
 
+
 ## Want to join?
 
 There are different ways to participate in this local effort:
 
- - You can include your GitHub users by adding the term “dominican” to your profile's location(dominicana is also accepted).
+p
```

Comment:
@savagemike I think that `p` should be removed. :)

Reply:
I did a new pull request, should be fixed. Let me know!